### PR TITLE
DevRel, Education and App Coordinator Service Proposal from Itering for 2025 Q1

### PR DESCRIPTION
**Pull request by boundless-forest**
This application aims to clearly outline the scope of the services Itering will provide in documentation, ecosystem education, and applications. We hope it will be well-received by community members. Feedback is welcome.

### CODE REVIEW

File: service-providers/itering/25Q1-doc-edu-app-coordinator.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+timeline: 2025-Q1
+costs: 12,000 USDC
+payment: 0x0678E7cB36D3b9A591dF7E42CA23074aF977C52B (Arbitrum One)
+---
+
+## Team
+
+### Member
+
+- [Bear Wang](https://github.com/boundless-forest)
```
Comment (hackfisher) [inline service-providers/itering/25Q1-doc-edu-app-coordinator.md:13]: Add Ran, Denny and other support from Itering.

File: service-providers/itering/25Q1-doc-edu-app-coordinator.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Building a strong and active community around RingDAO is essential. This service aims to bridge the gap between regular users and developers, ensuring a healthy and engaged community. The community education support includes:
+
+* Gathering and analyzing community feedback to improve products.
+* Collaborating with the marketing team and community members to organize online/offline events such as hackathons and AMAs.
+* Providing training and Q&A sessions for active community members to become community leaders, helping them organize and strengthen the community.
+
+### Application Coordination
+
+With the increasing number of applications built around RingDAO, it is crucial to ensure they are well-integrated, maintained, and updated. This service provides application coordination support, including:
```
Comment (hackfisher) [inline service-providers/itering/25Q1-doc-edu-app-coordinator.md:43]: Could be more specific about applications plans and milestones in Q1, such as para.link, XAccount.

Comment (boundless-forest) [reply service-providers/itering/25Q1-doc-edu-app-coordinator.md]: Done

File: service-providers/itering/25Q1-doc-edu-app-coordinator.md (hunk 1 above)
Comment (boundless-forest) [reply service-providers/itering/25Q1-doc-edu-app-coordinator.md:13]: Done

File: service-providers/itering/25Q1-doc-edu-app-coordinator.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Providing training and Q&A sessions for active community members to become community leaders, helping them organize and strengthen the community.
+
+### Application Coordination
+
+With the increasing number of applications built around RingDAO, it is crucial to ensure they are well-integrated, maintained, and updated. This service provides comprehensive application coordination support, including but not limited to:
+
+- ParaLink App: Launch the stable version of the ParaLink app and expand its user base from the RingDAO community to the broader Polkadot ecosystem.
+- RingDAO New Site: Oversee the development of the new RingDAO website, focusing on a refreshed look and improved user experience.
+- MsgPort New Site: Manage the development of the new MsgPort website, ensuring a modern design and enhanced user experience.
+- XAccount App: Coordinate and design the new version of the XAccount app.
```
Comment (hackfisher) [inline service-providers/itering/25Q1-doc-edu-app-coordinator.md:50]: Please add apps.darwinia.network maintenance/management and keep updated:

https://apps.darwinia.network/
https://github.com/darwinia-network/apps

**Review by hackfisher — APPROVED** [review]

**Pull request merged**